vspreview/toolbars/misc/toolbar.py

```diff
@@ -171,6 +171,25 @@ def add_shortcuts(self) -> None:
                 ViewMode.FFTSPECTRUM if self.main.current_viewmode != ViewMode.FFTSPECTRUM else ViewMode.NORMAL
             )
         )
+        self.main.add_shortcut(
+            Qt.Key.Key_Return, self.add_current_frame_to_comp
+        )
+
+    def add_current_frame_to_comp(self) -> None:
+        comp = self.main.toolbars.comp
+        if not comp.manual_frames_lineedit.isVisible():
+            return
+
+        frame = str(self.main.current_output.last_showed_frame)
+        current_frames = comp.manual_frames_lineedit.text()
+
+        if not current_frames:
```

Comment:
this check isn't needed, code below will generate an empty list and will just work

Reply:
No it won't 

```py
>>> "".split(",")
['']
```